### PR TITLE
Bump ember-cli-htmlbars to v5

### PR DESCRIPTION
**Pull request by nlfurniss**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "ember-cli-babel": "^7.18.0",
-    "ember-cli-htmlbars": "^4.2.3"
+    "ember-cli-htmlbars": "^5.3.1"
```
Comment (rwjblue) [inline package.json:30]: Can you move this from deps to devDeps? This addon does not have any templates in `addon/` or `addon-test-support/`.

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (nlfurniss) [reply package.json]: Done!

**Review by rwjblue — APPROVED** [review]
Looks good to me! 

@stefanpenner mind merge + release (I don't have access here)

**Comment by gabrielcsapo** [comment]
I can merge and publish!

**Review by gabrielcsapo — APPROVED** [review]

**Comment by gabrielcsapo** [comment]
@stefanpenner I can merge and release this

**Comment by gabrielcsapo** [comment]
published as https://github.com/stefanpenner/ember-strict-resolver/releases/tag/1.3.0

**Pull request merged**